packages/open-payments/src/client/index.ts

```diff
@@ -296,3 +339,42 @@ export const createAuthenticatedClient = async (
     })
   }
 }
+
+export const createMonetizationClient = async (
+  args: CreateMonetizationClientDeps
+): Promise<AuthenticatedClient> => {
+  const {
+    resourceServerOpenApi,
+    authServerOpenApi,
+    walletAddressServerOpenApi,
+    ...baseDeps
+  } = await createMonetizationClientDeps(args)
+
+  return {
+    incomingPayment: createIncomingPaymentRoutes({
+      ...baseDeps,
+      openApi: resourceServerOpenApi
+    }),
+    outgoingPayment: createOutgoingPaymentRoutes({
+      ...baseDeps,
+      openApi: resourceServerOpenApi
+    }),
+    walletAddress: createWalletAddressRoutes({
+      ...baseDeps,
+      openApi: walletAddressServerOpenApi
+    }),
+    grant: createGrantRoutes({
+      ...baseDeps,
+      openApi: authServerOpenApi,
+      client: args.walletAddressUrl
+    }),
+    token: createTokenRoutes({
+      ...baseDeps,
+      openApi: authServerOpenApi
+    }),
+    quote: createQuoteRoutes({
+      ...baseDeps,
+      openApi: resourceServerOpenApi
+    })
```

Comment:
should we consolidate this with the routes from `createAuthenticatedClient`? so that we don't end up missing to add a route accidentally in either place